src/Components/test/testassets/BasicTestApp/RouterTest/TestRouterWithAdditionalAssembly.razor

```diff
@@ -0,0 +1,11 @@
+@using Microsoft.AspNetCore.Components.Routing
+<Router AppAssembly="@typeof(BasicTestApp.Program).Assembly" AdditionalAssemblies="@(new[] { typeof(TestContentPackage.RouteableComponentFromPackage).Assembly, })">
```

Comment:
I think this is quite ugly, but tbh I don't have a better suggestion.

Reply:
You think the overall design is ugly? Or specifically the `@(new[] { typeof(TestContentPackage.RouteableComponentFromPackage).Assembly, })`?

If people want this to be more hygenic they can always place the values in a field and use `OnIntiailized`.